config/environments/test.rb

```diff
@@ -34,7 +34,7 @@
   config.action_controller.allow_forgery_protection = false
 
   # Store uploaded files on the local file system in a temporary directory.
-  config.active_storage.service = :test
+  config.active_storage.service = :public_google
```

Comment:
Does it need to be Google for the test environment?

Reply:
ah good point. I will admit theres no tests for these as yet. the settings were so unconventional!